cmd/lotus-storage-miner/info.go

```diff
@@ -33,6 +33,12 @@ var infoCmd = &cli.Command{
 	Subcommands: []*cli.Command{
 		infoAllCmd,
 	},
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "show-sectors-info",
+			Usage: "show sectors info",
```

Comment:
I'd make this flag inverted, so the default is to show the sectors (`hide-sectors`)

Reply:
updated